docs/2_step_by_step/2_2_experiment_and_run.md

```diff
@@ -1 +1,33 @@
-# Two basic concepts: Experiment and Run
\ No newline at end of file
+# Two basic concepts: Experiment and Run
+
+Finetuner organizes your training based on two concepts: **Experiment** and **Run**.
+
+An Experiment is defined as a general purpose machine learning task you're fine-tuning for.
```

Comment:
```suggestion
An Experiment defines the machine learning task you're fine-tuning for in general.
```
For me, a `general purpose ML task` would be something different, i.e., `classification` in general